### PR TITLE
Add management scripts to git and remove config.js files

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- Use env variables instead of injection to `config.js` files
- Remove main `config.js` file
- Update cluster management scripts in `/scripts` dir
- Disable `build` steps in _Makefiles_ (it's done in _Docker_ few minutes after)
- Disable `test` steps in _Dockerfiles_ (it's done in _Makefiles_ few minutes earlier)
- Update readme according to changes in `npm run bootstrap` command
- Change the way of importing `LuigiContextProvider` to match other imports
- Change Angular apps' buildpack to `@angular-builders/custom-webpack`

**Related issue(s)**

### CODE REVIEW

File: lambda/angular.json [hunk 1]
```diff
@@ -58,7 +63,7 @@
           }
         },
         "serve": {
-          "builder": "@angular-devkit/build-angular:dev-server",
+          "builder": "@angular-builders/dev-server:generic",
```
Comment (dariadomagala) [inline lambda/angular.json]: why this change?

File: lambda/src/app/event-activations/event-activations.service.spec.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ describe('EventActivationsService', () => {
     },
   ));
 
-  it('should get eventActivations', done => {
+  xit('should get eventActivations', done => {
```
Comment (dariadomagala) [inline lambda/src/app/event-activations/event-activations.service.spec.ts]: nope :)

File: lambda/src/styles.scss [hunk 3]
```diff
@@ -1,8 +1,11 @@
 // Include Fiori Fundamentals
-$fd-icons-path: '~fiori-fundamentals/scss/icons/';
-$fd-fonts-path: '~fiori-fundamentals/scss/fonts/';
-@import '~fiori-fundamentals/scss/all.scss';
-@import 'styles/fiori-fundamentals-overrides';
```
Comment (dariadomagala) [inline lambda/src/styles.scss]: what happened to our overrides?

File: lambda/tslint.json [hunk 4]
```diff
@@ -28,6 +46,7 @@
     "no-string-literal": false,
     "no-console": false,
     "ordered-imports": false,
+    "no-use-before-declare": false,
```
Comment (dariadomagala) [inline lambda/tslint.json]: why this change?

File: lambda/src/app/event-activations/event-activations.service.spec.ts (hunk 2 above)
Comment (parostatkiem) [reply lambda/src/app/event-activations/event-activations.service.spec.ts]: Yes. This test (and other deactivated ones) basically checks if there was a request on a certain domain. The domain in the test is always different from the actual one because env variables can't be injected into the build done for test purposes.

File: lambda/angular.json (hunk 1 above)
Comment (parostatkiem) [reply lambda/angular.json]: To inject the webpack config file into Angular.

File: lambda/tslint.json (hunk 4 above)
Comment (parostatkiem) [reply lambda/tslint.json]: _TSlint_ got a problem :(
I'm not sure if it's still valid though.

File: service-catalog-ui/Makefile [hunk 5]
```diff
@@ -2,8 +2,8 @@ IMG_NAME = service-catalog-ui
 IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(IMG_NAME)
 TAG = $(DOCKER_TAG)
 
-ci-pr: root-pr resolve test build pre-build-image pull-licenses build-image push-image
```
Comment (y-kkamil) [inline service-catalog-ui/Makefile]: it looks like we got rid of tests at all, as i see that they are also commented out in https://github.com/kyma-project/console/pull/1338/files#diff-053c242aa15f41912be5246bbbfc8b65R39

File: scripts/setClusterConfig.sh [hunk 6]
```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```
Comment (y-kkamil) [inline scripts/setClusterConfig.sh]: `scripts` folder contains files that have hyphen-separated names. this file has its name camelCased. i'd stick to one convention.

File: lambda/src/app/event-activations/event-activations.service.spec.ts (hunk 2 above)
Comment (y-kkamil) [reply lambda/src/app/event-activations/event-activations.service.spec.ts]: should we leave it like that then? or just remove?

File: scripts/setClusterConfig.sh (hunk 6 above)
Comment (parostatkiem) [reply scripts/setClusterConfig.sh]: It's done on purpose. Camelcase scripts are supposed to be run manually by the developer, while hyphen-separated ones are not.
If you still want, I can change it.

File: service-catalog-ui/Makefile (hunk 5 above)
Comment (parostatkiem) [reply service-catalog-ui/Makefile]: They were constantly failing anyway 🤣

File: README.md [hunk 7]
```diff
@@ -47,7 +47,7 @@ The Console also includes React and Angular libraries:
     > - builds all the libraries
     > - installs dependencies for all the [components](#components)
     > - updates your `/etc/hosts` with the `127.0.0.1 console-dev.kyma.local` host
-    > - ignores all `config.js` files in git
+    > - creates `.clusterConfig.gen` file if it doesn't exist (pointing at `kyma.local` domain)
```
Comment (kazydek) [inline README.md]:
```suggestion
    > - creates the `.clusterConfig.gen` file if it doesn't exist, pointing at the `kyma.local` domain
```
